benchmarks/src/it/scala/akka/kafka/benchmarks/Benchmarks.scala

```diff
@@ -125,25 +126,39 @@ class AlpakkaKafkaAtMostOnceConsumer extends BenchmarksBase() {
 }
 
 class ApacheKafkaPlainProducer extends BenchmarksBase() {
+  private val prefix = "apache-kafka-plain-producer"
+
   it should "bench with small messages" in {
-    val cmd = RunTestCommand("apache-kafka-plain-producer", bootstrapServers, 2000 * 1000, 100)
+    val cmd = RunTestCommand(prefix, bootstrapServers, 2000 * factor, 100)
     runPerfTest(cmd, KafkaProducerFixtures.initializedProducer(cmd), KafkaProducerBenchmarks.plainFlow)
   }
 
   it should "bench with normal messages" in {
-    val cmd = RunTestCommand("apache-kafka-plain-producer-normal-msg", bootstrapServers, 2000 * 1000, 5000)
+    val cmd = RunTestCommand(prefix + "-normal-msg", bootstrapServers, 2000 * factor, 5000)
+    runPerfTest(cmd, KafkaProducerFixtures.initializedProducer(cmd), KafkaProducerBenchmarks.plainFlow)
+  }
+
+  it should "bench with normal messages written to 8 partitions" in {
+    val cmd = RunTestCommand(prefix + "-normal-msg", bootstrapServers, 2000 * factor, 5000, numberOfPartitions = 8)
```

Comment:
The name of the command should be distinct from the one above. Add `-8-partitions`. This is important, because this name is used in the result output file name.

Reply:
Good catch!